text/0000-roadmap-2021.md

```diff
@@ -0,0 +1,133 @@
+
+### How does this group make decisions?
+
+This section establishes the process (or processes) by which the team makes decisions. This is useful for both team members and outside parties. Any normative documents produced by the team (for example, the Reference for lang team) should also be listed here.
+
+# Unifying processes and vocabulary
+
+Today, Rust teams decide individually where to host documents and have discussions. We do not want to take away these choices from the Rust teams: we believe that teams know best what works for them. However, sometimes the divergence here is unintentional, and providing defaults that work well can be an excellent way to remove friction.
+
+The specific details of the vocabulary and processes that we will seek to unify are not yet clear, primarily because there is no one place where all of the specifics of team activity are laid out. As we begin to establish charters over the course of the year, we will begin to identify common needs and current choices, and will work to diverge only when needed.
+
+Some examples of currently divergent process and vocabulary:
+
+* Project group vs. working group
+* Major change proposals in lang/compiler (but not other teams)
+* Best approach to propose new ideas differs across teams and is not
+  well-documented (and even within a team there can be lack of clarity, e.g., on
+  which things need a major change proposal or just a PR being fine)
+
+# Unified project tracking
+
+A frequent concern -- even amongst our most active team members -- is that knowing what is going on in teams can be difficult. It can be difficult to identify what efforts are active or temporarily paused, and what kind of feedback is sought for on proposals.
+
+As part of our charter and unified process/vocabulary work, we would like to work to surface the ongoing work in teams in a common way. The aim is to target contributors/maintainers, both existing and new, not people aiming to propose a new project. (It may be a useful gauge of how much bandwidth the team has left, though).
+
+However, it is expected that the road towards tracking team efforts for all of our teams is going to require more work than we can fit in this coming year. However, a number of our teams have already established or are working to establish similar tracking dashboards: the [language][lang-board], [library][libs-board], and [crates.io][crates-board] teams. We expect to unify the structure of these existing dashboards and publish guiding documentation which other teams can use as a framework for making their own; by the end of the year we want to add a compiler team dashboard but do not expect to be able to expand the set further.
+
+[lang-board]: https://github.com/rust-lang/lang-team/projects/2
+[libs-board]: https://github.com/rust-lang/libs-team/projects/2
```

Comment:
Not a comment on the RFC text, just a random thought..

This one seems geared towards keeping track of all proposals/ideas, not just ones being actively worked on. It sounds like we'll want to make that out of scope for the project trackers we're talking about here (especially since the size can be intimidating.) But teams can use separate boards for that purpose.